app/datasets.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
```

Comment:
The task was sent with the answers - we should keep them in another branch.
I think we should leave only return [] inside each function.